zyte_common_items/processors.py

```diff
@@ -330,6 +361,44 @@ def aggregateRating(self):
     return value
 
 
+def images_processor(value: Any, page: Any) -> Any:
+    """Convert the data into a list of :class:`~zyte_common_items.Image`
+    objects if possible.
+
+    If the input is a string, it's used as a url for returning image object.
+
+    If input is either an iterable of strings or mappings with "url" key, they are
+    used to populate image objects.
+
+    Other inputs are returned unchanged.
+    """
+
+    # TODO: add generic-purpose extract_images utility to zyte-parsers
+    #
+    # value = _handle_selectorlist(value)
+    # if isinstance(value, (Selector, HtmlElement)):
+    #    images = extract_images(value)
+    #    return [Image(url=url) for url in images]
```

Comment:
Could we move this comment into a repository issue instead?

Reply:
Removed comment in 12cb8efadd59d399b41bf59de6b0d721d4d10efb created issue https://github.com/zytedata/zyte-parsers/issues/16